website/settings/defaults.py

```diff
@@ -1958,3 +1964,8 @@ class CeleryConfig:
 DS_METRICS_BASE_FOLDER = None
 REG_METRICS_OSF_TOKEN = None
 REG_METRICS_BASE_FOLDER = None
+
+PUBLIC_PROJECT_STORAGE_CAP = 50
```

Comment:
We should move these two storage cap settings to the model work PR that @Johnetordoff is doing rather than having them in this PR.

Reply:
Removed 👍